tests/unit_tests/test_deplete_msr_continuous.py

```diff
@@ -52,34 +52,17 @@ def test_get_set(model):
     msr = MsrContinuous(op, model)
 
     # Test by Openmc material, material name and material id
-    mat, dest_mat = [m for m in model.materials if m.depletable]
-    for i in [mat, mat.name, mat.id]:
-        for j in [dest_mat, dest_mat.name, dest_mat.id]:
-            for key, value in removal_rates.items():
-                msr.set_removal_rate(i, [key], value, destination_material=j)
-                assert msr.get_removal_rate(i, key) == value
-                assert msr.get_destination_material(i, key) == str(dest_mat.id)
-            assert msr.get_elements(i) == removal_rates.keys()
-
-def test_msr(run_in_tmpdir, model):
```

Comment:
We could rename this `back_of_envelope_verification()`. Is there a reason it was removed?